superset/config.py

```diff
@@ -1282,6 +1282,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 GLOBAL_ASYNC_QUERIES_REDIS_STREAM_LIMIT_FIREHOSE = 1000000
 GLOBAL_ASYNC_QUERIES_JWT_COOKIE_NAME = "async-token"
 GLOBAL_ASYNC_QUERIES_JWT_COOKIE_SECURE = False
+GLOBAL_ASYNC_QUERIES_JWT_COOKIE_SAMESITE = None  # One of [None, 'None', 'Lax', 'Strict']
```

Comment:
nit: it's generally convention to add these as type annotations:
```suggestion
GLOBAL_ASYNC_QUERIES_JWT_COOKIE_SAMESITE: Optional[Literal['None', 'Lax', 'Strict']] = None
```